pyfolio/perf_attrib.py

```diff
@@ -522,3 +491,149 @@ def plot_risk_exposures(exposures, ax=None,
     ax.set_title(title)
 
     return ax
+
+
+def _truncate_and_warn(returns,
```

Comment:
Something like 'align' might be clearer here then 'truncate', to me truncate suggests something more like trimming the ends

Reply:
I agree, renamed to this